components/brave_wallet/browser/eth_nonce_tracker.cc

```diff
@@ -49,49 +61,18 @@ uint256_t GetHighestContinuousFrom(
   return highest;
 }
 
-}  // namespace
-
-EthNonceTracker::EthNonceTracker(EthTxStateManager* tx_state_manager,
-                                 JsonRpcService* json_rpc_service)
-    : tx_state_manager_(tx_state_manager),
-      json_rpc_service_(json_rpc_service),
-      weak_factory_(this) {}
-EthNonceTracker::~EthNonceTracker() = default;
-
-void EthNonceTracker::GetNextNonce(const EthAddress& from,
-                                   GetNextNonceCallback callback) {
-  const std::string hex_address = from.ToHex();
-  json_rpc_service_->GetTransactionCount(
-      hex_address, base::BindOnce(&EthNonceTracker::OnGetNetworkNonce,
-                                  weak_factory_.GetWeakPtr(), EthAddress(from),
-                                  std::move(callback)));
-}
-
-void EthNonceTracker::OnGetNetworkNonce(EthAddress from,
-                                        GetNextNonceCallback callback,
-                                        uint256_t network_nonce,
-                                        mojom::ProviderError error,
-                                        const std::string& error_message) {
-  if (!nonce_lock_.Try()) {
-    std::move(callback).Run(false, network_nonce);
+void EthNonceTracker::OnEthGetNetworkNonce(const std::string& from,
+                                           GetNextNonceCallback callback,
+                                           uint256_t result,
```

Comment:
original naming `network_nonce` is more clear.

Reply:
done